cmd/apichecker/apicheck.go

```diff
@@ -0,0 +1,1269 @@
+								if string(tokenizer.Text()) == "v2" {
+									case1 = true
+								}
+							}
+						case "n5":
+							inner := tokenizer.Next()
+							if inner == html.TextToken {
+								tempStr := string(tokenizer.Text())
+								if tempStr == "v3" {
+									case2 = true
+									resp = append(resp, tempStr)
+								}
+							}
+						case "n6":
+							inner := tokenizer.Next()
+							if inner == html.TextToken {
+								tempStr := string(tokenizer.Text())
+								if tempStr != "v4" {
+									case3 = true
+								}
+								if case1 && case2 && case3 {
+									return resp, nil
+								}
+								resp = append(resp, tempStr)
+							}
+						}
+					}
+				}
+			}
+		}
+	}
+	return resp, nil
+}
+
+// HTMLScrapeLocalBitcoins gets the check string for Yobit Exchange
+func HTMLScrapeLocalBitcoins(htmlData *HTMLScrapingData) ([]string, error) {
+	temp, err := http.Get(htmlData.Path)
+	if err != nil {
+		return nil, err
+	}
+	a, err := ioutil.ReadAll(temp.Body)
+	if err != nil {
+		return nil, err
+	}
+	r, err := regexp.Compile(htmlData.RegExp)
+	if err != nil {
+		return nil, err
+	}
+	str := r.FindString(string(a))
+	sha := crypto.GetSHA256([]byte(str))
+	var resp []string
+	resp = append(resp, crypto.HexEncodeToString(sha))
+	return resp, nil
+}
+
+// TrelloGetListsData gets required data for all the lists on the given trello board
+func TrelloGetListsData(idBoard string) ([]ListData, error) {
+	var resp []ListData
+	err := common.SendHTTPGetRequest(pathGetAllLists+idBoard+apiKey+apiToken, true, verbose, &resp)
+	if err != nil {
+		return resp, err
+	}
+	return resp, nil
+}
+
+// TrelloCreateNewCard creates a new card on the list specified on trello
+func TrelloCreateNewCard(fillData *CardFill) error {
+	params := url.Values{}
+	params.Set("idList", fillData.ListID)
+	if fillData.Name != "" {
+		params.Set("name", fillData.Name)
+	}
+	if fillData.Desc != "" {
+		params.Set("desc", fillData.Desc)
+	}
+	if fillData.Pos != "" {
+		params.Set("pos", fillData.Pos)
+	}
+	if fillData.Due != "" {
+		params.Set("due", fillData.Due)
+	}
+	if fillData.MembersID != "" {
+		params.Set("idMembers", fillData.MembersID)
+	}
+	if fillData.LabelsID != "" {
+		params.Set("idLabels", fillData.LabelsID)
+	}
+	_, err := common.SendHTTPRequest(http.MethodPost,
+		pathNewCard+params.Encode()+apiKey+apiToken,
+		nil,
+		nil)
+	return err
+}
+
+// TrelloCreateNewCheck creates a new checklist item within a given checklist from trello
+func TrelloCreateNewCheck(newCheck string) error {
+	params := url.Values{}
+	params.Set("name", newCheck)
+	_, err := common.SendHTTPRequest(http.MethodPost,
+		pathChecklists+trelloChecklistID+params.Encode()+apiKey+apiToken,
+		nil,
+		nil)
+	return err
+}
+
+// TrelloGetChecklistItems get info on all the items on a given checklist from trello
+func TrelloGetChecklistItems() (ChecklistItemData, error) {
+	var resp ChecklistItemData
+	path := fmt.Sprintf(pathChecklistItems, trelloChecklistID, configData.Key, configData.Token)
```

Comment:
This also assumes that trelloChecklistID is set, but doesn't account for if this is set in the persons config